src/test/kotlin/JiraPerformanceComparisonIT.kt

```diff
@@ -44,22 +50,48 @@ class JiraPerformanceComparisonIT {
         val results = listOf(baseline, experiment).map { it.get().prepareForJudgement(FullTimeline()) }
         FullReport().dump(
             results = results,
-            workspace = workspace.isolateTest("Compare")
+            workspace = currentTaskWorkspace.isolateTest("Compare")
         )
         dumpMegaSlowWaterfalls(results)
     }
 
+    @Test
+    fun shouldOnlyProcessGatheredData() {
+        val task = "2020-03-06T14-00-16.448"
```

Comment:
It won't work for me, because I don't have this data. It's fine to have this test, but it shouldn't be bound to the `comparePerformance` task. I'd extract it to a separate `IT`.

Reply:
Updated the code, moved all to separate test class and created separate gradle task for it